MAINTAINERS

```diff
@@ -46,6 +46,11 @@ R:	Ying-Chun Liu (PaulLiu) <paul.liu@linaro.org> [@grandpaul]
 S:	Maintained
 F:	core/arch/arm/plat-sunxi/
 
+AllWinner sun50i A64
+R:	Amit Singh Tomar <amittomer25@gmail.com>
```

Comment:
Would you please add your GitHub ID? BTW, is it @Atomar25 or @Amit-Radur ? Pick the appropriate one and add it in brackets.